torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5380,8 +5441,36 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    safe_casts_outputs=True),
 ]
 
+op_db += [
+    BinaryUfuncInfo('add',
```

Comment:
I think we already have opinfos for add and mul?

Reply:
`add` yes, `mul` no.